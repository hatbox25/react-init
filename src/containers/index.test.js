import React from 'react';
import { shallow } from 'enzyme';

import Main from './index';

describe('Main', () => {
  test('should render default page correctly', () => {
    const props = {
      location: {
        pathname: '/',
      },
    };

    const wrapper = shallow(<Main {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
