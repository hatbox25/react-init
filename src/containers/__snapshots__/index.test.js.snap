// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render default page correctly 1`] = `
<div>
  HOME
</div>
`;
